src/biome/text/modules/heads/classification/defs.py

```diff
@@ -112,10 +108,32 @@ def labels_with_probabilities(probabilities: torch.Tensor) -> Dict[str, float]:
         output.max_class = max_classes
         output.max_class_prob = max_classes_prob
 
+        # TODO: This is just a duplicate of output.max_class. Revise if it is still necessary!
         output.label = max_classes
         output.prob = max_classes_prob
 
-        return output
+        return output.as_dict()
```

Comment:
Just a minimal suggestion. Could we do the taskOutput -> dict transformation outside this method (we could do it in `PipelineModel.decode`.

We can loose control over the head output with dictionaries

Reply:
:+1: i like, Done